website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -248,6 +248,62 @@ The `policy` field for the node rule can have one of the following values:
 - `deny`: do not allow the resource to be read or modified. Deny takes
   precedence when multiple policies are associated with a token.
 
+## Node Pools rules
+
+Node pool rules are defined with a `node_pool` block. An ACL policy can
+include zero, one, or more node pool rules.
+
+<!-- TODO(luiz): add link to node pools API docs -->
+Node pool rule controls access to the Node Pool API such as create, update, and
+list node pools.
+
+Each node pool rule is labeled with the node pool name it applies to. You may
```

Comment:
Is omitting the label allowed so that it falls back to `"default"`, as we do for namespaces?

Reply:
After some internal discussions we're going to leave this one like this for now.